ui/src/style/pages/data-explorer/raw-text.scss

```diff
@@ -108,4 +108,13 @@ $raw-text-height: 38px;
   &.raw-text--success {
     color: $c-rainforest;
   }
-}
\ No newline at end of file
+}
+
+.dropdown.query-template {
+  position: absolute;
+  top: 8px;
+  right: 7px;
+  >.dropdown-toggle {
```

Comment:
This should just be `.dropdown-toggle`. The fact that it's nested inside `.dropdown.query-template` already means it's a descendant.